sklearn/covariance/tests/test_graphical_lasso.py

```diff
@@ -248,12 +249,71 @@ def test_graphical_lasso_cv_scores():
         X
     )
 
+    _assert_graphical_lasso_cv_scores(
+        cov=cov,
+        n_splits=splits,
+        n_refinements=n_refinements,
+        n_alphas=n_alphas,
+    )
+
+
+# TODO(1.5): remove in 1.5
+def test_graphical_lasso_cov_init_deprecation():
+    """Check that we raise a deprecation warning if providing `cov_init` in
+    `graphical_lasso`."""
+    rng, dim, n_samples = np.random.RandomState(0), 20, 100
+    prec = make_sparse_spd_matrix(dim, alpha=0.95, random_state=0)
+    cov = linalg.inv(prec)
+    X = rng.multivariate_normal(np.zeros(dim), cov, size=n_samples)
+
+    emp_cov = empirical_covariance(X)
+    with pytest.warns(FutureWarning, match="cov_init parameter is deprecated"):
+        graphical_lasso(emp_cov, alpha=0.1, cov_init=emp_cov)
+
+
+@pytest.mark.usefixtures("enable_slep006")
+def test_graphical_lasso_cv_scores_with_routing(global_random_seed):
```

Comment:
This is a test which checks when we route the groups parameter the graphical lasso cv still operates correctly and gives the desired results.

Reply:
I still don't see what this test brings on top of the common test we have

Reply:
This checks for the actual results and values to ensure the output is as desired after routing parameters.

Reply:
I think even if this test might be additional or extra maybe it would still be nice to have it since it compares the values? Let me know what you think @adrinjalali @glemaitre . Otherwise we can remove it to finalize this PR.